pkg/forwarder/forwarder.go

```diff
@@ -154,19 +157,21 @@ var _ Forwarder = &DefaultForwarder{}
 
 // Options contain the configuration options for the DefaultForwarder
 type Options struct {
-	NumberOfWorkers       int
-	RetryQueueSize        int
-	DisableAPIKeyChecking bool
-	KeysPerDomain         map[string][]string
+	NumberOfWorkers         int
+	RetryQueueSize          int
+	DisableAPIKeyChecking   bool
+	KeysPerDomain           map[string][]string
+	ConnectionResetInterval time.Duration
 }
 
 // NewOptions creates new Options with default values
 func NewOptions(keysPerDomain map[string][]string) *Options {
 	return &Options{
-		NumberOfWorkers:       config.Datadog.GetInt("forwarder_num_workers"),
-		RetryQueueSize:        config.Datadog.GetInt("forwarder_retry_queue_max_size"),
-		DisableAPIKeyChecking: false,
-		KeysPerDomain:         keysPerDomain,
+		NumberOfWorkers:         config.Datadog.GetInt("forwarder_num_workers"),
+		RetryQueueSize:          config.Datadog.GetInt("forwarder_retry_queue_max_size"),
+		DisableAPIKeyChecking:   false,
+		KeysPerDomain:           keysPerDomain,
+		ConnectionResetInterval: time.Duration(config.Datadog.GetInt("forwarder_connection_reset_interval")) * time.Second,
```

Comment:
in some extreme cases (mostly for QA/testing purposes, but there might be others...), we may want to set a value below one minute (for example 10 or 15 sec), in order to recreate connections for "almost" each new transaction.

It's debatable though. We could also use a duration-formatted string in the config file and grab it with `GetDuration` but we'd want to validate the value to avoid non-sensical ones (like `1ns`).

So overall I'm thinking seconds should meet all use cases.